astroid/__pkginfo__.py

```diff
@@ -35,8 +35,12 @@ def has_environment_marker_range_operators_support():
 
 if has_environment_marker_range_operators_support():
     extras_require[':python_version<"3.4"'] = ['enum34', 'singledispatch']
-elif py_version < (3, 4):
-    install_requires.extend(['enum34', 'singledispatch'])
+    extras_require[':python_version<"3.2"'] = ['functools32']
```

Comment:
Good point, thanks.  We had some conversation about it in IRC this afternoon, and I've been trying to prevent any more pull requests than we already have from languishing.